.github/workflows/publish-website.yml

```diff
@@ -16,6 +16,12 @@ jobs:
     - name: Checkout Repository
       uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
 
+    - name: Setup Gradle
```

Comment:
Commit message:
- "documentation in website" -> "documentation in(to) the website"